docs/writing_a_builder.md

```diff
@@ -74,6 +72,39 @@ the following noteworthy properties:
   If the builder had used `^proto/{{}}.proto` as an input, it would not have
   run on strict suffix matches.
 
+#### Using multiple groups
+
+A builder may use multiple capture groups in an input. Groups can be given a
```

Comment:
```suggestion
A builder may use multiple capture groups in an input. Groups must be given a
```

Reply:
I wasn't sure how to describe this since you can technically have one unnamed group (`{{}}/{{file}}.proto` is valid), but I agree that "must" fits better here :)